packages/dropdowns.legacy/demo/autocomplete.stories.mdx

```diff
@@ -22,7 +22,7 @@ import README from '../README.md';
 
 # API
 
-**DEPRECATED:** use `@zendeskgarden/react-dropdowns.next`
+**DEPRECATED:** use `@zendeskgarden/react-dropdowns`
 [Combobox](https://zendeskgarden.github.io/react-components/?path=/docs/packages-dropdowns-next-combobox--docs)
```

Comment:
Can we get these URLs fixed in prep for `main`? They'll be broken for now, but that's ok.

Reply:
Ah, yes. Can do. I'll open a separate PR for that.